.travis/install_bower.sh

```diff
@@ -1,5 +1,3 @@
-#!/bin/bash
-set -e
-set -x
+#!/bin/bash -ex
```

Comment:
I don't like this change. If we're going to use a shebang, I prefer `/usr/bin/env bash` and this also means that if I run `bash install_bower.sh` these options don't take effect.